apps/fluent-tester/src/TestComponents/RadioGroupExperimental/DefaultRadioGroup.tsx

```diff
@@ -0,0 +1,52 @@
+import { RadioGroup } from '@fluentui-react-native/experimental-radio-group';
+import { RadioButton } from '@fluentui/react-native';
+import { Separator } from '@fluentui/react-native';
+import * as React from 'react';
+import { View } from 'react-native';
+
+export const DefaultRadioGroup: React.FunctionComponent = () => {
+  // Client's example onChange function
+  const onChange = (key: string) => {
+    console.log(key);
+  };
+
+  const [selectedKey, setSelectedKey] = React.useState('C');
+
+  const onChange2 = React.useCallback((key: string) => {
+    if (key == 'A') {
```

Comment:
Let's just do:
`setSelectedKey(key)` here to improve perf

Reply:
Thanks for the suggestion, revised!